updating/updating-restricted-network-cluster.adoc

```diff
@@ -65,6 +65,19 @@ include::modules/images-configuration-registry-mirror.adoc[leveloffset=+2]
 
 include::modules/generating-icsp-object-scoped-to-a-registry.adoc[leveloffset=+2]
 
+[id="upgrading-rhel-nodes-restricted-network"]
+== Upgrading RHEL nodes on a restricted network
+
+The following procedures update {op-system-base-full} worker nodes on a restricted network.
+
+include::modules/obtaining-ocp-packages-restricted-network-cluster.adoc[leveloffset=+2]
+
+include::modules/prepare-populate-repo-server-restricted-network-cluster.adoc[leveloffset=+2]
+
+include::modules/populate-registry-restricted-network-cluster.adoc[leveloffset=+3]
+
+include::modules/rhel-compute-updating.adoc[leveloffset=+2]
```

Comment:
In this section, it is using subscription-manager to disable and enable proper repos, but this PR is on restricted network, subscription-manager is not available, and this is also why we introduce `obtaining-ocp-packages-restricted-network-cluster` and `prepare-populate-repo-server-restricted-network-cluster` to create a internal http yum repo to replace subscription-manager, right?